### ISSUE TITLE
Aggs support

### ISSUE THREAD

**Issue by laughp** [CottageLabs/facetview2#13]
Any plans to support aggs in the future? Seeing as to how facets will be deprecated in future ES versions. 

Thanks.

**Comment by emanuil-tolev** [comment]
Bad news - probably not: https://github.com/CottageLabs/facetview2/issues/8#issuecomment-115765360 . Good news: it's probably as simple as replacing "facets" with "aggs" for most common facet types.

So this was initially built because we made a lot of mistakes with facetview(1) and this is indeed much more extensible. Unfortunately a lot of things we want to do are still a pain in here like combining visualisation with facet drill-down, and we have started an even more general / abstracted project to deal with this: https://github.com/CottageLabs/edges .

Now, having said that, I just came back from an Elastic{ON} Tour day and Kibana 4 (which is an official part of the Elasticsearch stack being pushed by Elastic Co.) actually looks like it have all the querying and abstraction capability, as well as "widgeting" functionality - all things which edges has to do too, except then we have to maintain them. Kibana are very focussed on visualisation and charts and term facets are not quite as nice as the ones in facetview2. It remains to be seen whether we move towards contributing our functionality in facetview2/edges into Kibana or whether we continue developing edges.

So to recap: if your use case is to display some facets on the UI, edges does support aggregations and I'd encourage you to take a swing at it, but it is very young at this stage (but it might work almost out of the box too - depends on use case). If you're doing charting and vis, I would advise to look at customising Kibana as support over the long-term for the core functionality ought to be better as Elastic Co. has a lot more resources than us :). If that turns out to be impossible, edges does do charts (and mixes of aggs + vis) very well too.
